packages/gestalt-core/build.js

```diff
@@ -38,35 +35,11 @@ const svgPath = () => ({
   },
 });
 
-const statsPlugin = () => {
```

Comment:
Removed all the stats plugins since they accounted for 1-2 seconds for every incremental change